fedimint-server/src/lib.rs

```diff
@@ -113,6 +115,18 @@ pub async fn run(
     Ok(())
 }
 
+async fn update_server_info_version_dbtx(
+    dbtx: &mut DatabaseTransaction<'_>,
+    code_version_str: &str,
+) {
+    let mut server_info = dbtx.get_value(&ServerInfoKey).await.unwrap_or(ServerInfo {
+        init_version: code_version_str.to_string(),
+        last_version: code_version_str.to_string(),
+    });
+    server_info.last_version = code_version_str.to_string();
+    dbtx.insert_entry(&ServerInfoKey, &server_info).await;
+}
+
```

Comment:
If we instantly overwrite without triggering any check, is this DB entry worth anything?

Reply:
Well, the checks would go before. But we don't have any checks ATM. This is preparing it in case we ever need it.